app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -37,6 +37,29 @@
   <% end %>
 <% end %>
 
+<% if region.sanction_check_written? %>
+  <p class="govuk-body"><%= t("eligibility_interface.three_months_before_applying") %></p>
+<% end %>
+
+<% if region.status_check_online? || region.sanction_check_online? %>
+  <p class="govuk-body">
+    As your education department or authority has an online register of teachers,
+    you’ll also need to provide any reference numbers we need to check your record.
+  </p>
+<% end %>
+
+<% unless FeatureFlags::FeatureFlag.active?(:application_work_history) && !region.application_form_skip_work_history %> 
+  <p class="govuk-body"><%= proof_of_recognition_description_for(region:) %></p>
```

Comment:
I don't think we'd want this section to appear above the online status/sanction information aa it looks like the online sentence would now be broken up by the "work history" section.

Reply:
I'll move it above the sentence 👍